### PR TITLE
Simplify Github templates

### PR DESCRIPTION
**Pull request by timbot1789**
## This PR:

I find the formatting of our github templates hard to read, and I notice no one is really ever filling it out. I want to remove some unnecessary/redundant sections. This description is my proposed PR template.
 
**1.**    
**2.** If needed, delete if not    
**3.** If needed, delete if not

## Screenshots (if applicable):
Add any screenshots/videos here.

## Additional Context (optional):
Add any other context about the PR here.

## Future Steps/PRs Needed to Finish This Work (optional):
Add any other steps/PRs that may be needed to continue this work if this PR is just a step in the right direction.

## Issues needing discussion/feedback (optional):
**1.**    
**2.** If needed, delete if not    
**3.** If needed, delete if not

### CODE REVIEW

**Comment by xscottxbrownx** [comment]
Personally, I don't think the thin lines break up the sections enough - but I'm not married to the current template by any means. Not a big enough issue for me to argue. I'd be perfectly okay with removing the thicker lines.


Screenshot 2023-10-05 at 4 20 49 PM

But I think ⬆️ here shows how each section is clearly divided.  The automatic heading lines only underline the heading basically.

---

While we are at it, I vote to eliminate the `The Files This PR Effects` section - all of that info is found in the `Files changed` tab of the PR itself.

**Comment by timbot1789** [comment]
> Personally, I don't think the thin lines break up the sections enough - but I'm not married to the current template by any means. Not a big enough issue for me to argue. I'd be perfectly okay with removing the thicker lines.
> 
> Screenshot 2023-10-05 at 4 20 49 PM
> But I think ⬆️ here shows how each section is clearly divided. The automatic heading lines only underline the heading basically.
> 
> While we are at it, I vote to eliminate the `The Files This PR Effects` section - all of that info is found in the `Files changed` tab of the PR itself.

Idk maybe the issue I have is that no one is actually filling out any of these forms. At least for me, part of the reason is I see all these big thick lines and my mind goes "Ok stop here", and doesn't go further. 

Maybe a happy middle is removing / combining some of these sections. I'll put up another commit with your suggestion, Scott.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -1,54 +1,31 @@
...
----
 ### Actual Behavior:
 What actually happened, and how it differed from the expected behavior.
 
----
 ### Screenshots (If applicable):
 If possible, add screenshots to help explain the problem.
 
----
-### Desktop (please complete the following information):
```
Comment (timbot1789) [inline .github/ISSUE_TEMPLATE/bug_report.md:36]: I don't think we need these two sections. Right now, PASS is purely a browser application, so the desktop/smartphone fields aren't useful to our bug reports. The only thing that matters is the browser, and the only browser we support is the most recent version of chromium. Any bug report that isn't on the most recent version of chromium is immediately invalid.

If that changes, we can add this section back in with the configurations we support.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -1,54 +1,25 @@
----
-name: Bug report
```
Comment (timbot1789) [inline .github/ISSUE_TEMPLATE/bug_report.md:2]: These header sections are largely redundant information already contained in the issue metadata. Name is the title, about is the description, title is the title, so on.

**Comment by xscottxbrownx** [comment]
@timbot1789 not sure if this is technically ready for review or not, but I'm still okay with getting rid of the lines.

I also propose dropping the concise title sections - as we already type a title for the Issue or PR (we dropped it for PR template.)


Maybe we also adjust the github auto-template stuff (whatever you want to call it) at the top of the files, or drop them too...
Like the `Enhancement Request` Issue has a title of `Enhancement - [your title]`, and a label of `enhancement`.   Maybe we can do the same for Bug Report & Feature Request.

**Review by Jared-Krajewski — COMMENTED** [review]
Changes are fine with me. Scott and I were just talking about how these needed some attention so hopefully this help. I think we had even talked about removing the additional context section  and just leaving the description as the one source to provided info. I also think we can safely remove the title headers since they are already the title of the issue once its created.
 ``` 
### Bug Title:
Please provide a CONCISE description of the bug.
``` 
and 
```
### Feature Title:
Please provide a CONCISE description of the proposed feature.
```

File: .github/ISSUE_TEMPLATE/bug_report.md
Comment (Jared-Krajewski) [inline .github/ISSUE_TEMPLATE/bug_report.md:1]: I think actual behavior is very close to the bug description and could go as well.

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -1,54 +1,31 @@
...
 **1.**    
 **2.** If needed, delete if not    
 **3.** If needed, delete if not
 
----
 ### Expected Behavior:
 What should have happened if the bug didn't occur.
 
----
 ### Actual Behavior:
```
Comment (Jared-Krajewski) [inline .github/ISSUE_TEMPLATE/bug_report.md:21]: I think actual behavior is very close to the bug description and could go as well.

Comment (Jared-Krajewski) [inline .github/ISSUE_TEMPLATE/bug_report.md]: I think actual behavior is very close to the bug description and could go as well.

Comment (timbot1789) [reply .github/ISSUE_TEMPLATE/bug_report.md]: removed

**Comment by timbot1789** [comment]
@xscottxbrownx @Jared-Krajewski responded to all your comments. I think this is gtg.

**Review by Jared-Krajewski — APPROVED** [review]
🚀

**Pull request merged**